package.json

```diff
@@ -24,6 +24,7 @@
   "author": "Simon Ihmig <ihmig@kaliber5.de>",
   "license": "MIT",
   "devDependencies": {
+    "@types/node": "^10.12.10",
```

Comment:
Is that intentional?

Reply:
just a DX improvement for anyone working with node stuff in this package